app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -387,9 +387,12 @@ You can select by class:
 ```html
 <dom-module id="x-foo">
   <template>
-    <style>
-      ::slotted(.green) {
-        color: green;
+    <style> 
+      <!-- To work within the Shady CSS scoping shim limitations,
+           and to ensure consistent cross-browser behavior,
+	   add a selector to the left of the ::slotted(.classname) notation. --> 
```

Comment:
Note that this applies to _any_ `::slotted()` selector, so it should be moved up to the main discussion of `::slotted()`, and shown in all of the examples.